client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -86,5 +121,55 @@ def make_ndarray(tensor_proto):
         >>> output = make_ndarray(tensor_proto)
         >>> print(output)
     '''
+    tensor_shape = [d.size for d in tensor_proto.tensor_shape.dim]
```

Comment:
What if tensor_proto is invalid (does not contain shape, dtype or even data)? 
Is it possible to prepare such tensor_proto? If yes, we could have a method to validate if tensor_proto contains everything we need to make ndarray from it.